examples/go/tagging/handler/handler.go

```diff
@@ -8,12 +8,13 @@ import (
 	"log"
 	"net/http"
 	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
 
+	"github.com/codegold79/vsphere/vsClient"
```

Comment:
There's some issues in this pkg which could affect your fn code. E.g. Client construction could lead to half-initialized client, where the client is not correctly initialized but also != nil so your code would fail at a later stage. (hard to troubleshoot and test). Consider providing the client pkg as a separate pkg in this fn and not external dependency so it can be tested and reviewed together with this fn.

Reply:
Per our discussion, I have moved client back to the `function` package, but in a different file than `handler.go`.